client/active-game/active-game-manager.js

```diff
@@ -117,10 +119,21 @@ export default class ActiveGameManager extends EventEmitter {
     if (current && current.config) {
       this.emit('gameCommand', gameId, 'routes', routes)
       this.emit('gameCommand', gameId, 'setupGame', current.config.setup)
-      this.emit('gameCommand', gameId, 'allowStart')
     }
   }
 
+  allowStart(gameId, waitTime = 2000) {
```

Comment:
Not quite clear what the wait time here is intended to accomplish, is it meant to be like a minimum amount of time to show the loading screen for? Perhaps we should keep that value elsewhere so we're sure it's the same for all clients if so (passed from the server? or at least stored in a common constant somewhere?)

Could even consider structuring this such that the 'allowStart' is triggered directly from the server for all clients (that is, the server emits another event when it's okay for clients to start)

Reply:
Yeah, the original intention of wait time was to show the loading screen for a minimum amount of time. I didn't want to make it a constant because I thought it might be possible for different gameplay activities to display a loading screen in a different way.

Your point about keeping it same for all clients stands though; and I actually initially planned on making the server in charge of dealing with it, but decided against it as this seemed more straightforward and I didn't see any downsides. But yeah, I'll move it to the server since it makes more sense for server to keep all clients in sync about starting a game.